cpp-package/include/mxnet-cpp/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Rebuildable file(s)
+op.h
```

Comment:
You mean to build libmxnet.so?  I would expect so, yes.  The problem I encountered is that this doesn;t all normally get built, so when I went ot build it, it didn't build at all, and I spent 3 days fixing it.  At least by building all or part of it as part of the build, we can ensure that the main codebase doesn't diverge again.  It's a quick build (cpp-package)
wdyt?